input/pagecontent/overview.md

```diff
@@ -188,6 +188,9 @@ For more information on this transaction, see [Transactions - Creating and respo
 #### Updating CarePlan and CareTeam
 The CarePlan Service is responsible for updating the CareTeam and, for convenience, the CarePlan.activities. This transaction is triggered by a Task creation or update at the CP-Service. 
 
+The CP-Service evaluates the Task update (is state transition allowed?). The Task state determines if the Task.owner should or shouldn't be a participant in the CareTeam (see the table below). Once a Task is completed or failed, the Task.owner remains a participant, but the (active) period is ended. Active CareTeam.participants are authorized to access patient data at other CareTeam.participants. Inactive (period is ended) CareTeam.participants can access the Tasks, CarePlan and CareTeam. Other Care providers can just access their Tasks. (for more info see [security-authorization](authorization.html))
```

Comment:
(out of scope for this PR)
Maybe we should have a terminology section at the top, so we can use CPC and CPS as abbrv.